packages/adblocker-benchmarks/blockers/ublock.js

```diff
@@ -6,165 +6,62 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-const { URL } = require('url');
-const punycode = require('punycode');
-const fs = require('fs');
-const path = require('path');
+const modulePromise = import('ubo-snfe');
 
-const SandboxedModule = require('sandboxed-module');
-const { JSDOM } = require('jsdom');
```

Comment:
I think we can clean-up the `package.json` as well and remove the following two dependencies:
- `jsdom`
- `sandboxed-module`

Reply:
Done.